openlibrary/plugins/openlibrary/js/tabs.js

```diff
@@ -1,4 +1,5 @@
 const TABS_OPTIONS = { fx: { opacity: 'toggle' } };
+import 'jquery-ui/ui/widgets/tabs';
 
 export default function initTabs($node) {
```

Comment:
Ah, I think we need this to be non-default to reference it as `module.initTabs`

```suggestion
export function initTabs($node) {
```